src/test/facility.test.js

```diff
@@ -85,6 +85,32 @@ describe('Facility route endpoints', () => {
       expect(response.body.data).to.have.property('amenities');
     });
   });
+  describe('PATCH /api/facility/supplier/:supplierId/:roomId', () => {
+    it('should successfully update a room category - 201', async () => {
```

Comment:
What about the test case for when the amenity/room does not exist and when the updater does not have the required roles?

Reply:
I'll update the tests swiftly.